build/linter/bootstrap/analyzer.go

```diff
@@ -89,20 +90,44 @@ func run(pass *analysis.Pass) (interface{}, error) {
 						continue
 					}
 				case v.Tok == token.CONST && len(v.Specs) > 1:
-					lastSpec := v.Specs[len(v.Specs)-1]
-					v2, ok := lastSpec.(*ast.ValueSpec)
-					if !ok {
-						continue
-					}
-					if len(v2.Names) != 1 {
-						continue
-					}
-					name := v2.Names[0].Name
-					maxVerVariable, err = strconv.Atoi(name[len("version"):])
-					if err != nil {
-						continue
+					for _, spec := range v.Specs {
+						v2, ok := spec.(*ast.ValueSpec)
+						if !ok {
+							continue
+						}
+						if len(v2.Names) != 1 {
+							continue
+						}
+						name := v2.Names[0].Name
+						if !strings.HasPrefix(name, "version") {
+							continue
+						}
+
+						valInName, err := strconv.Atoi(name[len("version"):])
+						if err != nil {
+							continue
+						}
+
+						if len(v2.Values) != 1 {
+							panic(fmt.Sprintf("bootstrap.go: the value of version variable '%s' must be specified explicitly", name))
```

Comment:
we can use `pass.Reportf` like below lines. It can report the line/column number of golang AST